### PR TITLE
Indexing domain theory papers

### PR DESCRIPTION
**Pull request by tomdjong**

### CODE REVIEW

**Comment by tomdjong** [comment]
@martinescardo I finished indexing Part I, cleaning up a few bits in the process and recording the universal properties of the product and exponential of dcpos in Brendan's files.

File: source/DomainTheory/Basics/Curry.lagda [hunk 1]
```diff
@@ -211,6 +213,46 @@ module _ (𝓓 : DCPO {𝓤} {𝓤'})
                y
                p
 
+\end{code}
+
+Added 3 July 2024 by Tom de Jong.
+
+\begin{code}
+
+ -- We introduce abbreviations for readability
```
Comment (martinescardo) [inline source/DomainTheory/Basics/Curry.lagda:222]: Can this comment be outside the code environment?

Comment (tomdjong) [reply source/DomainTheory/Basics/Curry.lagda:222]: Sure. I'll make it part of the new PR I'm about to merge.

File: source/DomainTheory/Part-I.lagda [hunk 2]
```diff
@@ -0,0 +1,819 @@
...
+
+\begin{code}
+
+{-# OPTIONS --safe --without-K --lossy-unification #-}
+
+open import UF.FunExt
+open import UF.Subsingletons
+open import UF.PropTrunc
+
+-- Our global assumptions
```
Comment (martinescardo) [inline source/DomainTheory/Part-I.lagda:22]: Can this comment be outside the code environment?

File: source/DomainTheory/Part-I.lagda [hunk 3]
```diff
@@ -0,0 +1,819 @@
...
+
+Definition-2-3 : (𝓥 : Universe) (X : 𝓤 ̇  ) → 𝓥 ⁺ ⊔ 𝓤 ̇
+Definition-2-3 𝓥 X = 𝓟 {𝓥} X
+
+Definition-2-4 : (𝓥 : Universe) (X : 𝓤 ̇  )
+               → (X → 𝓟 {𝓥} X → 𝓥 ̇  )
+               × (𝓟 {𝓥} X → 𝓟 {𝓥} X → 𝓥 ⊔ 𝓤 ̇  )
+Definition-2-4 𝓥 X = _∈_ , _⊆_
+
+{- Section 3 -}
```
Comment (martinescardo) [inline source/DomainTheory/Part-I.lagda:66]: In general, I would prefer all comments to be outside code environments.

File: source/DomainTheory/Part-I.lagda [hunk 4]
```diff
@@ -0,0 +1,819 @@
...
+             → is-prop (has-sup _⊑_ α)
+  Remark-3-8 = having-sup-is-prop _⊑_
+
+module _ (𝓥 : Universe) where
+ open import DomainTheory.Basics.Dcpo pt fe 𝓥
+
+ Definition-3-9 : {𝓤 𝓣 : Universe} → (𝓤 ⊔ 𝓥 ⊔ 𝓣) ⁺ ̇
+ Definition-3-9 {𝓤} {𝓣} = DCPO {𝓤} {𝓣}
+
+ -- Remark-3-10: No formalisable content.
```
Comment (martinescardo) [inline source/DomainTheory/Part-I.lagda:125]: Is this a meta-mathematical remark?

Comment (tomdjong) [reply source/DomainTheory/Part-I.lagda:125]: Yes, Remark 3.10 just says that it's important to keep track of universe levels.

Comment (martinescardo) [reply source/DomainTheory/Part-I.lagda:125]: Maybe add this sentence in the comment, then?

File: source/DomainTheory/Part-I.lagda [hunk 5]
```diff
@@ -0,0 +1,819 @@
...
+             × Φ ∘ Ψ ∼ id
+             × 𝓓∞ ≃ᵈᶜᵖᵒ (𝓓∞ ⟹ᵈᶜᵖᵒ 𝓓∞)
+Theorem-8-16 = ε-exp∞-section-of-π-exp∞ ,
+               π-exp∞-section-of-ε-exp∞ ,
+               𝓓∞-isomorphic-to-its-self-exponential
+
+Remark-8-17 : Σ σ₀ ꞉ ⟨ 𝓓∞ ⟩ , σ₀ ≠ ⊥ 𝓓∞⊥
+Remark-8-17 = σ₀ , 𝓓∞⊥-is-nontrivial
+
+\end{code}
```
Comment (martinescardo) [inline source/DomainTheory/Part-I.lagda:819]: Excellent. I like this style of summarizing the formalization to match the paper. It reads well.

**Review by martinescardo — COMMENTED** [review]
Excellent. I don't have further comments.

**Comment by tomdjong** [comment]
Thanks!

**Pull request merged**